### PR TITLE
[bitnami/nginx] stream server blocks

### PR DESCRIPTION
**Pull request by TrueGameover**
### Description of the change

Configuration allowed for stream server blocks in nginx container.

### Benefits

Can configure stream server blocks.

### Possible drawbacks

no

### Additional information

https://github.com/bitnami/containers/pull/72353

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by jotamartos — DISMISSED** [review]
Changes LGTM, just a minor suggestion

**Review by jotamartos — CHANGES_REQUESTED** [review]
Please bump the version of the Chart, there was a new release in the meantime.

File: bitnami/nginx/templates/deployment.yaml [hunk 1]
```diff
@@ -245,6 +248,10 @@ spec:
             {{- if .Values.extraEnvVars }}
             {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
+            {{- if or .Values.streamServerBlock .Values.existingStreamServerBlockConfigmap }}
+            - name: NGINX_ENABLE_STREAM
+              value: "true"
+            {{- end }}
```
[the remark this answers is not in the document]
Comment (TrueGameover) [reply bitnami/nginx/templates/deployment.yaml]: ok, I changed it

File: bitnami/nginx/Chart.yaml [hunk 2]
```diff
@@ -34,4 +34,4 @@ maintainers:
 name: nginx
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/nginx
-version: 18.1.15
+version: 18.1.16
```
Comment (jotamartos) [inline bitnami/nginx/Chart.yaml:37]: As you are adding new parameters, we need to use a new minor version

```suggestion
version: 18.2.0
```

**Review by jotamartos — APPROVED** [review]

**Pull request merged**